### PR TITLE
Update import statements and import files that other files depend on

### PR DESCRIPTION
**Pull request by ckiefer14**
Import errors in hopp/simulation pythons files
base.py works in other locations

### CODE REVIEW

**Review by jaredthomas68 — CHANGES_REQUESTED** [review]

File: hopp/simulation/technologies/hydrogen/electrolysis/run_h2_distributed.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/simulation/technologies/hydrogen/electrolysis/run_h2_distributed.py:1]: run_h2_distributed is an exact duplicate. The other location is in simulation/technologies/Electrolyzer_Models

File: hopp/simulation/technologies/hydrogen/electrolysis/run_h2_power_powerElec.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/simulation/technologies/hydrogen/electrolysis/run_h2_power_powerElec.py:1]: run_h2_power_powerElec is an exact duplicate. The other location is in simulation/technologies/Electrolyzer_Models.

This location was chosen over the other as this directory would be empty if these files were deleted from /Electrolyzer_Models/

File: hopp/to_organize/to_organize2/LCA_single_scenario.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/LCA_single_scenario.py:1]: This has history associated with it

File: hopp/to_organize/to_organize2/LCA_single_scenario_ProFAST.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/LCA_single_scenario_ProFAST.py:1]: This file has history with it

File: hopp/to_organize/to_organize2/h2_optimize_gf.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/h2_optimize_gf.py:1]: History is with the file

File: hopp/to_organize/to_organize2/hopp_for_h2_floris.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/hopp_for_h2_floris.py:1]: History is still there

File: hopp/to_organize/to_organize2/pipelineASME.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/pipelineASME.py:1]: History is with file

File: hopp/to_organize/to_organize2/plot_battery.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/plot_battery.py:1]: History is with file

File: hopp/to_organize/to_organize2/plot_power_to_load.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/plot_power_to_load.py:1]: History is here with file

File: hopp/to_organize/to_organize2/plot_reopt_results.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/plot_reopt_results.py:1]: History is with file

File: hopp/to_organize/to_organize2/plot_shortfall_curtailment.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/plot_shortfall_curtailment.py:1]: History is with file

File: hopp/to_organize/to_organize2/run_profast_for_ammonia.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/run_profast_for_ammonia.py:1]: History is with file

File: hopp/to_organize/to_organize2/run_profast_for_h2_transmission.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/run_profast_for_h2_transmission.py:1]: History is with file

File: hopp/to_organize/to_organize2/run_profast_for_hydrogen.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/run_profast_for_hydrogen.py:1]: History is with file

File: hopp/to_organize/to_organize2/run_profast_for_steel.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/run_profast_for_steel.py:1]: History is with file

File: hopp/to_organize/to_organize2/run_reopt.py
[the remark this answers is not in the document]
Comment (ckiefer14) [reply hopp/to_organize/to_organize2/run_reopt.py:1]: History is with file

**Pull request merged**